README.md

```diff
@@ -16,68 +16,75 @@ To get started with the demo app, please ensure Flutter is installed on your sys
-- Add the required Kotlin code to this file to this file, usually within the `configureFlutterEngine` method. 
-```
-override fun configureFlutterEngine(flutterEngine: FlutterEngine) {
-    super.configureFlutterEngine(flutterEngine)
-    // Your Kotlin code here...
-}
-```
+#### 3. Updating `AndroidManifest.xml` with `meta-data` (if required):
+- Within the `android/app/` directory, locate and open the `AndroidManifest.xml` file.
+- If the mediated ad network adapter you add requires `meta-data`, insert the necessary `meta-data` elements within the `<application>` tag. 
 
 ### iOS 
 #### 1. Adding adapters to your podfile:
 - Locate your `Podfile` in the `/ios` folder.
-- Open the Podfile with a text editor or IDE.
-- Add the adapter pods to your application as specified in the [documentation](https://developers.applovin.com/en/flutter/preparing-mediated-networks#ios). It will look something like this
+- Open the `Podfile` with a text editor or IDE.
+- Add the necessary adapter pods for the mediated networks you're integrating, as specified in the [documentation](https://developers.applovin.com/en/flutter/preparing-mediated-networks#ios). It will look something like this
 ```
 target 'Runner' do
   flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
-  pod `AppLovinSDK`
+  pod 'AppLovinMediationExampleAdapter'
 end
 ```
+
+> [!CAUTION]
+> Do not add the underlying AppLovin SDK to your dependencies. The AppLovin MAX Flutter plugin
+> already specifies the certified SDK version corresponding to the plugin. Manually adding it may
+> break your build and cause unpredictable results.
+>
+> ~~pod 'AppLovinSDK'~~
+
 #### 2. Install the Pods:
-- Save the `Podfile`.
-- Open a terminal window, navigate to the root of your Flutter project, and run the following command to install the pods:
-```flutter pub get```
-- Alternatively, you may also run `pod install` within the `ios` directory. 
+- After saving the `Podfile`, open a terminal and run the following commands to install the pods:
+```
+pod install --repo-update
+```
 
-#### 3. Adding code to your Xcode Project (if required):
+#### 3. Updating the Bundle Identifier with your package name:
 - In your Flutter project's `ios` folder, find and open the `Runner.xcworkspace` file to launch Xcode.
-- Locate the `AppDelegate.m` (Objective-C) or `AppDelegate.swift` (Swift) file within Xcode.
-- Add the necessary code in the `didFinishingLaunchingWithOptions` as the adapter documentation directs. 
+- Select the Signing and Capabilities tab and update the Bundle Identifier with your package name, ensuring it matches the package name configured in your AppLovin dashboard.
+
+#### 4. Adding the `NSUserTrackingUsageDescription` key to the Information Property List (Info.plist):
+- Add `NSUserTrackingUsageDescription` to your Information Property List via Xcode or directly edit
+  `Info.plist`. This key is required to display the permission prompt for tracking user activity across apps.
```

Comment:
The second sentence is a little ambiguous. It at first glance reads as though the key must display the prompt. I think you mean something like "This key is necessary so that [your app / MAX / the plugin / something] can display the permission prompt..." In any case, [adding a subject](https://docs.google.com/document/d/1mvaE1khnBVsZA30Mau_S4bfjbqyB9uxoPXEDN45hopY/edit#heading=h.w4angfuaeym9) to the verb "display" should fix the ambiguity.